docs/userguide.md

```diff
@@ -239,6 +239,10 @@ select count(*) from account;
 1 row selected (1.97 seconds)
 ```
 
+## TiSparkR
+A thin layer build for supporting R language with TiSpark
```

Comment:
It'd be better to make it a complete sentence: "TiSparkR is a thin layer build for supporting the R language with TiSpark."